compiler-rt/lib/scudo/standalone/secondary.h

```diff
@@ -174,9 +174,9 @@ template <typename Config> class MapAllocatorCache {
       if (!Entry.isValid())
         continue;
       Str->append("StartBlockAddress: 0x%zx, EndBlockAddress: 0x%zx, "
-                  "BlockSize: %zu\n",
+                  "BlockSize: %zu %s\n",
```

Comment:
I know this is a really, really small nit, but could this be modified slightly so there is no whitespace at the end?

For example:

BlockSize: %zu%s\n", ..., EntryTime == 0 ? " [R]" : "");

Reply:
Sorry, I was under the impression that we have to have non-empty string in ScopedString for the placeholder and I was wrong.

Fixed